tests/html/mut_observer_perf.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset='utf-8'>
+</head>
+<body>
+    <p id="pid1" style='color:red;'>Paragraph tag</p>
+    <p id="pid2" style='color:red;'>Paragraph tag</p>
+<script>
+window.onload = function(){
+    var start_test = function(targetNode) {
+        var start = Date.now();
```

Comment:
We should switch to `performance.now()` instead of `Date.now()` for more precise times.
Same with `var stop`.